peps/pep-0101.rst

```diff
@@ -562,6 +562,11 @@ the main repo.
   - Make sure the ``SOURCE_URI`` in ``Doc/tools/extensions/pyspecific.py``
     points to ``main``.
 
+  - Update the version numbers for the Windows builds in ``PC/`` and
+    ``PCbuild/``, which have references to ``python38``::
```

Comment:
Is `python38` part of an overall example? Or should we fix up that wording? (I'm sure the RM will figure it out, it just kind of jumps out as "why?")

Reply:
Yes, for this local section.

There's a nearby earlier bit like:

> First, set main up to be the next release, i.e.X.Y+1.a0:
```console
$ git checkout main
$ .../release-tools/release.py --bump 3.9.0a0
```
And a nearby later bit like:

> Commit these changes to the main branch:
```console
$ git status
$ git add ...
$ git commit -m 'Bump to 3.9.0a0'
```

But then elsewhere we also have more generic ones:

> Tag the release for 3.X.YaN:
```console
$ .../release-tools/release.py --tag 3.X.YaN
```

And specific ones like:

```
$ curl -X PURGE https://www.python.org/ftp/python/3.12.0/Python-3.12.0.tar.xz
```

We can probably figure out which RM added each bit :)

I think it's fine for now, they can be cleaned up all together at some other time.